pet.py

```diff
@@ -22,20 +22,21 @@ def stop():
 
 @cli.command()
 @click.argument('name')
-@click.option('--templates', '-t', multiple=True, default=None, help='Creates project using templates')
+@click.option('--templates', '-t', multiple=True, help='Creates project using templates')
 def create(name, templates):
     """creates new project"""
     output = bl.create(name, templates)
     if output:
         click.secho(output, fg='red')
 
 
-@cli.command()
-def list():
+@cli.command('list')
+@click.option('--tasks', '-t', help="List available tasks in project")
```

Comment:
make it subcommand for project:
`pet project_name list` 
above command will list all tasks for `project_name`